src/Microsoft.TestPlatform.ObjectModel/Client/Interfaces/IDebugTestHostLauncher.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading;
+
+namespace Microsoft.VisualStudio.TestPlatform.ObjectModel.Client.Interfaces
+{
+    /// <summary>
+    /// Interface defining contract for custom test host implementations
+    /// </summary>
+    public interface IDebugTestHostLauncher : ITestHostLauncher
```

Comment:
Sorry for revisiting this. But did we decide to remove Debug from the name? Don't remember what we decided finally :)

There is an IsDebug flag on the base type, so I was wondering again whether its best to just call this ITestHostLauncher2 and allow implementers to decide whether or not they want to support the attach behavior...

Reply:
We decided to leave Debug in the name, but in the light of your newest point I agree we should change it back to `ITestHostLauncher2`. Best for symmetry as well.